.github/workflows/ci.yml

```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on: [push]
+
+jobs:
+  cleanup-runs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: rokroskar/workflow-run-cleanup-action@master
```

Comment:
@rokroskar maybe you can move this one to the renku repo or somewhere in the SDSC orgM

Reply:
I'm not sure it's necessary but yes, could do that at some point. sdsc can also fork it.